### DISCUSSION TITLE
NotFoundError in Autocomplete

### DISCUSSION THREAD

**Discussion by rhurlbatt** [anvilistas/anvil-extras#254] [Q&A] [answered]
I have just had the following error occur a couple of times intermittently today. Wondering if anyone can point me in the right direction.

`ExternalError: NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
at app/anvil_extras/Autocomplete/__init__.py:220   
called from app/anvil_extras/routing/_routing.py:150
called from app/anvil_extras/routing/_routing.py:536`

The line that called this was

`routing.set_url_hash(f'edit-asset?id={self.item["uuid"]}',
                         full_width_row=True,
                         item = item_copy,
                        )`

line 220 in `__init__.py`  refers to the hide event but I am not sure why there would be an error - I am trying to recreate this but haven't had any luck.

**Comment by s-cork** [comment, marked as answer]
Maybe replace line 220 with
`self._lp_node.remove()`

**Comment by Tiepies** [comment]
I am encountering the same error, so I will apply this patch, but I was wondering: @s-cork Did your answer imply a suggestion to fork and submit a PR with this change? If so, I'm happy to do that.

**Reply by s-cork** [reply]
That would be great.
